cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -93,7 +97,6 @@ var (
 		regexp.MustCompile("Reservation (.*) is incorrect for the requested resources"),
 		regexp.MustCompile("Zone does not currently have sufficient capacity for the requested resources"),
 		regexp.MustCompile("Reservation (.*) does not have sufficient capacity for the requested resources."),
```

Comment:
shouldn't this one also go under RESERVATION_CAPACITY_EXCEEDED?

Reply:
I was talking about it with @mtrqq -- this is the old error message, it's left here in case of rollbacks. However, I think we should remove that line soon.

Reply:
Actually, I'll remove it now.